sample/src/main/java/io/ona/kujaku/sample/activities/BaseLayerSwitcherActivity.java

```diff
@@ -31,7 +32,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         kujakuMapView = findViewById(R.id.kmv_baseLayerSwitcher_mapView);
         kujakuMapView.onCreate(savedInstanceState);
-        kujakuMapView.focusOnUserLocation(true);
+        kujakuMapView.focusOnUserLocation(true, RenderMode.COMPASS);
```

Comment:
Hey, so this shows the arrow facing the north and not the direction that the user is facing. What is the experience on testing it, does it show the direction you face such that you can easily navigate to wherever you wanna go? Try **GPS** render mode which `shows the device location with the icon bearing updated from the Location updates being provided to the LocationComponent` and test to see if this is correct

Reply:
![Screen Shot 2019-07-17 at 1 42 54 PM](https://user-images.githubusercontent.com/31766075/61369458-d2676e80-a898-11e9-9572-82bb6c23d4cc.png)

I rechecked the estimates file ☝️. It says to use a compass approach & direction user is facing, what do you guys think?

Reply:
I've tried the compass one and it is well working on my personnal phone. It shows the direction you face and then the direction wherever you wanna go. 
I can try it on other phones here but the Compass approach do the job.

Reply:
Yea true, tried it too and it works OK. 🤷‍♂ Weird that the documentation says otherwise. I will test it out in the evening on my way home